src/index.js

```diff
@@ -2,9 +2,14 @@ import 'bootstrap/dist/css/bootstrap.min.css';
 import './style.css';
 import completeTask from './checkbox';
 import addTask from './addtodo';
+import editTodo from './edit';
+import deleteTodo from './delete';
+import clearTodo from './clearChecked';
```

Comment:
- Well Done implementing all the functions in separate `js` files. :smile:

Reply:
thankyou, how can i fix the link on live demo?